boot/cypress/keys/enc-ec256-priv.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Does this need to be a separate copy of the same key that is in the root directory of the project? Can the build files just refer to that file instead of another copy?

Reply:
yes, you're right - it should refer to common folder